src/jit/jiteh.cpp

```diff
@@ -1075,7 +1075,7 @@ void* Compiler::ehEmitCookie(BasicBlock* block)
 
     void* cookie;
 
-#if FEATURE_EH_FUNCLETS && defined(_TARGET_ARM_)
+#if FEATURE_EH_FUNCLETS && defined(_TARGET_ARM_) && defined(LEGACY_BACKEND)
     if (block->bbFlags & BBF_FINALLY_TARGET)
```

Comment:
Can you explain what you've changed in the "finally target bit" handling for RyuJIT such that this is not required for RyuJIT/ARM32?